app/views/components/_option-select.html.erb

```diff
@@ -1,6 +1,5 @@
 <% title_id = "option-select-title-#{title.parameterize}" %>
 <div class="app-c-option-select"
-  <% if options.any? { |option| option[:data_attributes] } %>data-module="track-click"<% end %>
```

Comment:
Why is it necessary to remove this attribute from options in the select input? Does that mean that `uncheck_track_category: "filterRemoved",` won't be applied on this element?

Reply:
This line added `data-module='track-click'` if there is a `data_attributes` key in the hash (which is used for Google analytics tracking). There is a listener that looks for events in the DOM with this attribute and will send the necessary GA tracking event on that change. The checkbox now fires its own GA events (so if this line were left in we'd get two GA events) so it's necessary to remove this data attribute

Reply:
Ah makes sense, thanks for explaining.